src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
```

Comment:
The class name 'Rectangle' is appropriate as it is a noun, but ensure that no verbs are used for class names in your project.